examples/platformvm/importTx-avax-to-the-P-Chain-from-the-C-Chain-and-consume-a-multisig-atomic-output-and-a-create-multisig-output.ts

```diff
@@ -0,0 +1,117 @@
+import { Avalanche, BinTools, BN, Buffer } from "avalanche/dist"
+import {
+  PlatformVMAPI,
+  KeyChain,
+  SECPTransferOutput,
+  SECPTransferInput,
+  TransferableOutput,
+  TransferableInput,
+  UTXOSet,
+  UTXO,
+  AmountOutput,
+  UnsignedTx,
+  Tx,
+  ImportTx
+} from "avalanche/dist/apis/platformvm"
+import {
+  PrivateKeyPrefix,
+  DefaultLocalGenesisPrivateKey,
+  Defaults
+} from "avalanche/dist/utils"
+
+const ip: string = "localhost"
+const port: number = 9650
+const protocol: string = "http"
+const networkID: number = 1337
+const avalanche: Avalanche = new Avalanche(ip, port, protocol, networkID)
```

Comment:
InitAvalanche pls.

Reply:
for some reason github shows an outdated version of the file. see also https://github.com/chain4travel/caminojs/pull/74/files/97867e0d86ea1121ac82e3a52060102f90cda171..485cf701cbe93eb43d10e160dc9a79bff7242e22#diff-59f8144299dcb9f8ad83af72e0e99382d145d796d5fab51c9ffd41c54c1d1b74